File: help/pt_pt/exercicios_da_ficha_11/Ficha11_012/Ficha11_012.cpp
// Programador           -

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha11_012 {

    public:
        int main() {
            cout << "Altura : ";
            int altura;
            cin >> altura;
            cout << "Caractér : ";
            string c;
            cin >> c;
            trianguloC(altura , c ) ;
            return 0;
        }

        //::::::::::::::::::::::::::::::::::::::::::::::::::
        //User Defined Functions Code
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void linhaC( int n, string c ){
            int i  = 0 ;
            for( int i = 0; i < n; i = i + 1 ) {
                cout << c ;
            }
            cout << "\n";
        }
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void trianguloC( int altura, string c ){
            int i  = 0 ;
            for( int i = 1; i < altura + 1; i = i + 1 ) {
                linhaC(i , c ) ;
            }
        }
};// end of class Ficha11_012
// Execute main function
int main(){
    Ficha11_012 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:12

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_012/Ficha11_012.cs
// Programador           -

using System.IO;
using System;
public class Ficha11_012 {

    //Main Function
    static void Main() {
        Console.Write("Altura : " );
        int altura = int.Parse(Console.ReadLine());
        Console.Write("Caractér : " );
        String c = Console.ReadLine();
        trianguloC(altura , c ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linhaC( int n , String c){
        for( int i = 0; i < n; i = i + 1 ) {
            Console.Write(c );
        }
        Console.Write("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void trianguloC( int altura , String c){
        for( int i = 1; i < altura + 1; i = i + 1 ) {
            linhaC(i , c ) ;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:12
